libc/hdr/math_macros.h

```diff
@@ -21,7 +21,6 @@
 #if defined(__GNUC__) && !defined(__clang__)
 #define _GLIBCXX_INCLUDE_NEXT_C_HEADERS
 #endif
-#include <math.h>
```

Comment:
`<math.h>` should be included in the overlay mode.

Reply:
can you explain me why it is needed in overlay mode ?

Reply:
If you look at other proxy headers in `libc/hrd/` folder, the structure is that if it is in full build mode (detected with `#ifdef LIBC_FULL_BUILD`, then we include our own headers in `libc/include/llvm-libc-*`.  OTOH, if it is in overlay mode, we will just use the corresponding system header.  And in this case, the corresponding system header is `<math.h>`.

Reply:
The main purpose of proxy headers in `libc/hdr` is to provide a clear separation between our own public headers and the system headers, make sure that they won't be mixed and mismatched in different build modes.